### ISSUE TITLE
Cross-domain generalization test results

### ISSUE THREAD

**Issue by niuge12138** [ztsrxh/DVANet#1]
Hello! Your paper is good. 
But when I test the cross-domain generalization, I can't get results similar to table 6 in your paper. In the description of table 6 in your paper, you said "All methods are trained on the Scene Flow and tested on the train set of KITTI 2015." So I use the pretrained model you provided and test its performance on the train set of KITTI 2015 with the test script **test.py**. The results I got are quite different with table 6. 
The following is the information output by the test script, 
`vali: EPE:4.932, D1:22.673,  >1px:55.536, >3px:22.936 | wrde:25.441 | rmse:10.699, a1:0.029, rel:0.810`
I want to know if there is something wrong with my testing steps? Thanks.

**Comment by ztsrxh** [comment]
Thanks for your interest in our work. 
We have carefully reviewed the experiment records and uploaded the correct weights. This weight produces the results reported in paper. Specific values may vary slightly depending on experiment environments and hardware versions. 
The cross-domain capability of the model is not satisfying because of the depth-aware attention module. The depth patterns vary severely between different datasets. We are concentrating on this problem recently.

**Comment by niuge12138** [comment]
Thank you for your reply. 
I just used your latest Sceneflow pre-training weights to test directly on the KITTI2015 training set. The results look improved compared to the previous weights, but still did not get the accuracy in Table 6 in the paper.
`vali: EPE:2.352, D1:11.246,  >1px:40.032, >3px:11.480 | wrde:19.228 | rmse:10.138, a1:0.135, rel:0.703
`
Could you elaborate on how you test cross-domain generalization performance? Are the weights you use only trained on the Sceneflow dataset? Thank you.
